builds/fccsw-build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+#---Create stampfile to enable our jenkins to purge old builds------------------------------
+touch $WORKSPACE/controlfile
+
+#---Set up environment----------------------------------------------------------------------
+cd $WORKSPACE/fccsw
+source $WORKSPACE/fccsw/init.sh
+#source /cvmfs/fcc.cern.ch/sw/views/releases/0.9.0/x86_64-slc6-gcc62-opt/setup.sh
```

Comment:
You can remove this line - I updated the init script in the TrickTrack Pr to use this setup script!

Reply:
Do you mean the comment?

Reply:
Yes.